ASPNetCoreMastersTodoList.Api/Filters/ItemValidationFilterAttribute.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Mvc.Filters;
+using Repositories;
+using Services;
+using Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace ASPNetCoreMastersTodoList.Api.Filters
+{
+    public class ItemValidationFilterAttribute : Attribute, IActionFilter
+    {
+        public void OnActionExecuted(ActionExecutedContext context)
+        {
+            
+        }
+
+        public void OnActionExecuting(ActionExecutingContext context)
+        {
```

Comment:
The filter should be able to handle if the itemID is unavailable for the request, Thanks!

Reply:
updated